### DISCUSSION TITLE
How to get icon of weather?

### DISCUSSION THREAD

**Discussion by nchistov** [open-meteo/open-meteo#449] [Q&A]
This API returns weathercode, but I can't find the weather icon.

For example, https://www.weatherapi.com/ supports weather icon:

```json
...

"condition": {
            "text": "Partly cloudy",
            "icon": "//cdn.weatherapi.com/weather/64x64/day/116.png",
            "code": 1003
        },

...
```

But https://www.weatherapi.com/ isn't for open source developers.

So, how to get the icon of the weather?

**Comment by patrick-zippenfenig** [comment]
There are no official ICONs form Open-Meteo. You can use the `weathercode` variable and any weather ICON database. You might have to match the filename of each ICON with the `weathercode` yourself
